taxi/tests/test_forms.py

```diff
@@ -0,0 +1,62 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm, DriverLicenseUpdateForm, CarForm
+from taxi.models import Driver, Manufacturer
+
+
+class FormsTests(TestCase):
+    def test_driver_creation_form_with_license_number(self) -> None:
+        form_data = {
+            "username": "new_user",
+            "first_name": "Test Name",
+            "last_name": "Test Last",
+            "password1": "HardPassword12345",
+            "password2": "HardPassword12345",
+            "license_number": "ABC12345",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)

```

Comment:
The assertion `self.assertEqual(form.cleaned_data, form_data)` might fail if the form adds or modifies any data during the cleaning process. Consider checking only the relevant fields or using `form.cleaned_data.get('field_name')` for comparison.